docs/reference/errors-and-warnings/NU5127.md

```diff
@@ -11,9 +11,9 @@ f1_keywords:
 
 # NuGet Warning NU5127
 
-<pre>This package does not contain a lib/ or ref/ folder, and will therefore be treated as compatible for all frameworks. Since framework specific files were found under the build/ directory for net45, netstandard2.0, consider creating the following empty files to correctly narrow the compatibility of the package:
--lib/net45/_._
--lib/netstandard2.0/_._</pre>
+> This package does not contain a lib/ or ref/ folder, and will therefore be treated as compatible for all frameworks. Since framework specific files were found under the build/ directory for net45, netstandard2.0, consider creating the following empty files to correctly narrow the compatibility of the package:
+> -lib/net45/_._
+> -lib/netstandard2.0/_._
```

Comment:
```suggestion
> This package does not contain a lib/ or ref/ folder, and will therefore be treated as compatible for all frameworks. Since framework specific files were found under the build/ directory for net45, netstandard2.0, consider creating the following empty files to correctly narrow the compatibility of the package: <br/>
> -lib/net45/_._ <br/>
> -lib/netstandard2.0/_._
```